styles.css

```diff
@@ -0,0 +1,181 @@
+/* Variable Declaration */
+:root {
+  /* General */
+  --font-family: "Inter", sans-serif;
+  --big-font: 40px;
+  --medium-font: 24px;
+  --small-font: 12px;
+  --paragraph-font: 20px;
+  --main-bgc: #e5e5e5;
+  --primary-bgc: #fff;
+  --main-clr: #36b37e;
+  --primary-clr: #172b4d;
+  --secondary-clr: #3a4a42;
+  --neutral-clr: #42526e;
+  --box-width: 327px;
+
+  /* Hamburger / Nav */
+  --hamburger-clr: #28352f;
+  --nav-bgc: #fff;
+  --nav-clr: #3a4a42;
+}
+
+/* General Reset */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+ul[role="list"],
+ol[role="list"] {
+  list-style: none;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+li {
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  padding: 7px;
+}
+
+.wrapper {
+  width: 375px;
+}
+
+body {
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+  line-height: 1.5;
+  font-family: var(--font-family);
+  text-align: center;
+  align-items: center;
+  margin: 0 auto;
+}
+
+/* end of general styles */
+
+/* Main */
+.header {
+  display: flex;
```

Comment:
- [ ] You are repeating `display: flex` 4 times, and this bad practice, since you are repeating yourself, I would suggest you create a class for ex: `d-flex` with `display: flex` and give it to the elements that you want to be displayed as flex